demo/src/main/webapp/js/Aladin.js

```diff
@@ -184,10 +184,16 @@ const Aladin = (function () {
       zoomDiv.appendChild(zoomMinus);
       aladinDiv.appendChild(zoomDiv);
 
-      zoomPlus.addEventListener("click", () => aladin.increaseZoom())
+      zoomPlus.addEventListener("click", (e) => {
```

Comment:
How was this done on the old jquery version?

Reply:
jQuery was using this piece of code, just for the zoomPlus, it is the same idea for the zoomMinus
`var zoomPlus = $(aladinDiv).find(".zoomPlus");
  zoomPlus.click(function () {
    aladin.increaseZoom();
    return false;
});`

Anyways, the submit event it is being triggered in both codes with and without jQuery, but in the react-aladin, even if the submit event was triggered, it has no effect over the render. For explore is different, a submit triggers a re-render or something similar and that's the reason why it the problem is seen there.
Also I've tested with previous commits in explore (before jQuery was removed and even before react-aladin was modified to a functional component and the problem was still there).